src/components/Editor.tsx

```diff
@@ -1,6 +1,26 @@
 "use client";
-import MonacoEditor from "@monaco-editor/react";
+import MonacoEditor, { type Monaco } from "@monaco-editor/react";
+
+function configureEditor(monaco: Monaco) {
+  monaco.languages.json.jsonDefaults.setDiagnosticsOptions({
+    enableSchemaRequest: true,
+    validate: true,
+    schemas: [
+      {
+        uri: "https://gist.githubusercontent.com/jbottigliero/e7276bbdb5e56974e49ea69f91660c4a/raw/690956b1102341232741e8aeb1e7ce80adc7d1cd/flow_definition_schema.json",
```

Comment:
Could this be added to `/public/` as `/public/schemas/flow_definition_schema.json` then use a relative reference? `uri: "/schemas/flow_definition_schema.json"`

Reply:
Yeah probably if that's preferable. I'll try it

Reply:
Yep works fine!